--- origsrc/libffi-3.0.6/Makefile.am	2008-02-21 07:36:18.000000000 -0600
+++ src/libffi-3.0.6/Makefile.am	2008-09-03 15:30:32.635434400 -0500
@@ -165,7 +165,7 @@
 
 AM_CFLAGS = -Wall -g -fexceptions
 
-libffi_la_LDFLAGS = -version-info `grep -v '^\#' $(srcdir)/libtool-version`
+libffi_la_LDFLAGS = -version-info `grep -v '^\#' $(srcdir)/libtool-version` -no-undefined
 
 AM_CPPFLAGS = -I. -I$(top_srcdir)/include -Iinclude -I$(top_srcdir)/src
 AM_CCASFLAGS = $(AM_CPPFLAGS)
